File: data/src/main/java/com/data/net/retrofit/RetrofitService.java
package com.data.net.retrofit;

import io.reactivex.Flowable;
import retrofit2.http.GET;

import static com.data.net.DataLayerConstants.GET_PRIVACY_POLICY_AGREEMENT_URL;

public interface RetrofitService {
    @GET(GET_PRIVACY_POLICY_AGREEMENT_URL)
    Flowable<BaseResponse<String>> getPrivacyPolicyUrl();
}

File: data/src/main/java/com/data/net/retrofit/RetrofitService.kt
package com.data.net.retrofit

import com.data.GET_PRIVACY_POLICY_AGREEMENT_URL
import com.data.net.pojo.User
import io.reactivex.Flowable
import retrofit2.http.Body
import retrofit2.http.POST

interface RetrofitService {

    @POST(GET_PRIVACY_POLICY_AGREEMENT_URL)
    fun register(@Body map: Map<String, String?>?): Flowable<BaseResponse<User?>?>

    @POST(GET_PRIVACY_POLICY_AGREEMENT_URL)
    fun login(@Body map: Map<String, String?>?): Flowable<BaseResponse<User?>?>
}
